docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-create-custom-post-plugin.md

```diff
@@ -0,0 +1,95 @@
+* [Sending requests with Data Exchange API](/docs/pbc/all/data-exchange/{{page.version}}/sending-requests-with-data-exchange-api.md)
+
+
+## Create a post plugin
+
+In some cases, you need to perform some actions after importing or changing data through the Data Exchange API.
+
+For some specific requirements, you may need to create custom post plugins. Post plugins allow you to perform some manipulations after data import or data update. 
+To implement custom post plugins, you need to create a new class that implements the `Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostCreatePluginInterface` or `Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostUpdatePluginInterface` interface.
+
+You can see more details in already existing plugins, please check documentation [Install the Data Exchange API + Category Management feature](/docs/pbc/all/data-exchange/{{page.version}}/install-and-upgrade/install-the-data-exchange-api-category-management-feature.html) or [Install the Data Exchange API + Inventory Management feature](/docs/pbc/all/data-exchange/{{page.version}}/install-and-upgrade/install-the-data-exchange-api-inventory-management-feature.html).
+
+
+### Implement the `DynamicEntityPostCreatePluginInterface` interface
+
+For example, we will create a plugin that will call a facade method after data import for a specific entity, it's an abstract example, and you can implement your logic. If you need to implement a plugin for data updates, you need to implement the  `DynamicEntityPostUpdatePluginInterface` interface. 
+Methods `postCreate` and `postUpdate` should return `DynamicEntityPostEditResponseTransfer` object, you can add some errors in this transfer object if needed. `DynamicEntityPostEditRequestTransfer` object to get data from the request. Please see the example below.
+
+**Code sample:**
+
+```php
+<?php
+
+namespace Pyz\Zed\CustomModule\Communication\Plugin\DynamicEntity;
+
+use Generated\Shared\Transfer\DynamicEntityPostEditRequestTransfer;
+use Generated\Shared\Transfer\DynamicEntityPostEditResponseTransfer;
+use Propel\Runtime\Collection\Collection;
+use Spryker\Zed\DynamicEntityExtension\Dependency\Plugin\DynamicEntityPostCreatePluginInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+class CustomDynamicEntityPostCreatePlugin extends AbstractPlugin implements DynamicEntityPostCreatePluginInterface
+{
+    protected const TABLE_NAME = 'pyz_custom_table';
+
+    public function postCreate(DynamicEntityPostEditRequestTransfer $dynamicEntityPostEditRequestTransfer): DynamicEntityPostEditResponseTransfer
+    {
+        if ($dynamicEntityPostEditRequestTransfer->getTableName() !== static::TABLE_NAME) {
+            return new DynamicEntityPostEditResponseTransfer();
+        }
+
+        $responseTransfer = $this->getFacade()->createSubTestEntity($dynamicEntityPostEditRequestTransfer->getRawDynamicEntities()->getArrayCopy());
```

Comment:
If we call some facade method, it should exist. I would recommend to just add some basic functionality to the plugin, instead of using facade method.